src/Device.ts

```diff
@@ -80,6 +80,13 @@ export function detectDevice(): BuiltinHandlerName | undefined
 		const browser = Bowser.getParser(ua);
 		const engine = browser.getEngine();
 
+    const browserInfo = Bowser.parse(window.navigator.userAgent)
```

Comment:
There is no window in Node and mediasoup-client works in Node. You are using spaces instead of tab, so breaking `npm run lint` task.

Please read the docs properly. You can choose DeviceName via API.

Reply:
Sorry, you are right: https://mediasoup.org/documentation/v3/mediasoup-client/api/#Device-dictionaries

I didn't find that solution before.